src/Lecture1.hs

```diff
@@ -50,7 +51,8 @@ Explanation: @sumOfSquares 3 4@ should be equal to @9 + 16@ and this
 is 25.
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-sumOfSquares x y = error "TODO!"
+sumOfSquares :: Num a => a -> a -> a
+sumOfSquares x y = (+) ((*) x x) ((*) y y)
```

Comment:
That's a lot of `()` 😅 
I guess, you have Lisp or Clojure background 🙂

Reply:
I love prefix notation though never used it(I have change with Haskell)
Don't have Lisp background. Most of the time work with erlang/elixir